### DISCUSSION TITLE
Installation Help

### DISCUSSION THREAD

**Discussion by c-korifidis** [medusajs/medusa#9899] [Q&A]
Hello everybody.

I am struggling to install meduse for 6 hours but no luck.
After few hours of searching I figured out that 
npx create-medusa-app@latest
command is working with http 1.0 otherwise you must be lucky not to an error about incomplete download.
If I understand correct it has something to do with encoding that http 1.0 does not do but this is ok now.

Now I am facing this issue.

While I installation to this point:

🚀 Starting project setup, this may take a few minutes.
✔ Created project directory
✔ Database medusa-market created
✔ Installed Dependencies
✔ Project Built

The next step tells me:
An error occurred while preparing project: Error: Command failed: npx medusa db:migrate

With no other clue. 
I don't  even know where to look for the error.

Any help please!

**Comment by princessfrombabylon99** [comment]
[Check this blog](https://pevey.com/medusa-server-on-ubuntu/2-configuring-the-server)

Use the following command to install node v-20
`sudo apt install curl && \
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh | bash && \
source ~/.profile && \
nvm install 20`
